channels.go

```diff
@@ -39,14 +39,12 @@ type Channel struct {
 	Creator            string         `json:"creator"`
 	IsArchived         bool           `json:"is_archived"`
 	IsGeneral          bool           `json:"is_general"`
-	IsGroup            bool           `json:"is_group"`
-	IsStarred          bool           `json:"is_starred"`
```

Comment:
I am pretty sure this is part of a Channel structure. Are you sure we can remove this?